project/AkkaBuild.scala

```diff
@@ -100,14 +100,18 @@ object AkkaBuild {
       scalacOptions in Compile ++= (
         if (JavaVersion.isJdk8)
           Seq("-target:jvm-1.8")
-        else
+        else if (!CrossJava.Keys.useSystemJdk.value) {
```

Comment:
That's a lot of negations - would this become easier to read if we switched some of the logic around?

Reply:
I could name it `useJdk8` or something like that and default to `true`. But since it's only one negation and it's kind of an override away from the default behaviour I would prefer it this way.

Reply:
I agree with @raboof on clarity, this seems overly convoluted. Cleaner is better.

Reply:
Tried to improve clarity a bit